docs/admin/admission-controllers.md

```diff
@@ -387,6 +387,32 @@ This plug-in will deny any pod that attempts to set certain escalating [Security
 This plug-in implements automation for [serviceAccounts](/docs/user-guide/service-accounts).
 We strongly recommend using this plug-in if you intend to make use of Kubernetes `ServiceAccount` objects.
 
+### PersistentVolumeClaimResize
```

Comment:
Please help keep the list of builtin admission controllers ordered by names, i.e. insert this before `PersistentVolumeLabel`.

Reply:
done